kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -1006,9 +1006,99 @@ selected(active) - dark black
 
 .desktop-area {
     /*-fx-background-image: url("journal/grid-2x2-segment.png");*/
-    -fx-background-color: #828d9fff;
+    -fx-background-color: transparent; /* #828d9fff */
 }
 
+.desktop-scroll-pane {
+    -fx-background-color: #cad3e2; /* #cad3e2 */
+    -fx-background-insets: 0;
+    -fx-padding: 0 -12 -12 0;
+}
+
+.desktop-scroll-pane > .viewport {
+    -fx-background-color: transparent;
+    -fx-padding: 0;
+}
+
+.desktop-scroll-pane > .corner {
+    -fx-background-color: transparent;
+}
+
+.desktop-scroll-pane > .scroll-bar {
+    -fx-background-color: transparent;
+}
+
+.desktop-scroll-pane > .scroll-bar:vertical {
+    -fx-min-width: 0;
+    -fx-pref-width: 0;
+    -fx-max-width: 0;
+    -fx-padding: 3 3 3 -12;
+}
+
+.desktop-scroll-pane > .scroll-bar:horizontal {
+    -fx-min-height: 0;
+    -fx-pref-height: 0;
+    -fx-max-height: 0;
+    -fx-padding: -12 3 3 3;
+}
+
+.desktop-scroll-pane > .scroll-bar .track {
+    -fx-background-color: transparent;
+}
+
+.desktop-scroll-pane > .scroll-bar:vertical .track {
+    -fx-background-insets: 0 12 0 0;
+}
+
+.desktop-scroll-pane > .scroll-bar:horizontal .track {
+    -fx-background-insets: 0 0 12 0;
+}
+
+.desktop-scroll-pane > .scroll-bar .thumb {
+     -fx-background-color: -Grey-1, rgba(0, 0, 0, 0.4);
+     -fx-background-insets: 0, 1;
+     -fx-background-radius: 6px;
+    -fx-padding: 0;
+}
+
+.desktop-scroll-pane > .scroll-bar:hover .thumb {
+    -fx-background-color: -Grey-1, rgba(0, 0, 0, 0.5);
+}
+
+.desktop-scroll-pane > .scroll-bar .increment-button,
+.desktop-scroll-pane > .scroll-bar .decrement-button {
+    -fx-background-insets: 0;
+    -fx-min-width: 0;
+    -fx-pref-width: 0;
+    -fx-min-height: 0;
+    -fx-pref-height: 0;
+    -fx-padding: 0;
+}
+
+.desktop-scroll-pane > .scroll-bar .increment-arrow,
+.desktop-scroll-pane > .scroll-bar .decrement-arrow {
+    -fx-shape: " ";
+    -fx-padding: 0;
+}
+
+/*.scroll-pane.scroll-area > .scroll-bar:horizontal,*/
+/*.scroll-pane.scroll-area:focused > .scroll-bar:horizontal {*/
+/*    -fx-pref-height: 196px;*/
+/*    -fx-background-color: -Grey-3;*/
+/*    -fx-background-insets: 0;*/
+/*    -fx-background-radius: 0;*/
+/*    -fx-padding: 0;*/
+/*}*/
+
+/*.scroll-pane.scroll-area > .viewport,*/
+/*.scroll-pane.scroll-area:focused > .viewport {*/
+/*    -fx-pref-height: 196px;*/
+/*    -fx-background-color: -Grey-3;*/
+/*    -fx-background-insets: 0;*/
+/*    -fx-background-radius: 0;*/
+/*    -fx-padding: 0;*/
+/*}*/
+
```

Comment:
Can we remove commented out code?

Reply:
Thanks for catching that! I’ll remove the commented-out code right away.